addons/Makefile

```diff
@@ -13,9 +13,9 @@ export GO111MODULE := on
 # Image URL to use all building/pushing image targets
 IMG_DEFAULT_NAME ?= addons-controller
 IMG_DEFAULT_TAG := $(shell git describe --always --dirty --tags)
-IMG_DEFAULT_NAME_TAG := $(IMG_DEFAULT_NAME):$(IMG_DEFAULT_TAG)
-
```

Comment:
Is the idea that one can override the tag with IMG_VERSION_OVERRIDE. Nit: does IMG_TAG_OVERRIDE make more sense? Feel free to ignore.

The rest of the modified lines are rather confusing
Do you think it will be a little clearer with something like

```
IMG_TAG = $(IMG_TAG_OVERRIDE)
IMG_TAG ?= $(IMG_DEFAULT_TAG)
IMG_DEFAULT_NAME_TAG := $(IMG_DEFAULT_NAME):$(IMG_TAG)
```

Reply:
Ack. Removing this change from this PR as this might need to be changed on the `main` branch.